testing/framework/TestSCons_time.py

```diff
@@ -38,18 +38,18 @@
 import sys
 
 def write_args(fp, args):
-    fp.write(args[0] + '\\n')
+    fp.write(args[0] + os.linesep)
```

Comment:
Will this be an issue with windows where linesep != \n?

Reply:
I don't believe so, I think it makes it "better" because this is a "tool" which mocks behavior of some command, and if you were to just run that tool it would behave as expected with the correct line endings. But I've been wrong before, we'll see if there's any complaint from the appveyor build.